### PR TITLE
Add top holders percent of total supply metric

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: src/api/queries/blockchainQueries.js [hunk 1]
```diff
@@ -242,3 +242,22 @@ ApiClient_.prototype.fetchEthSpentOverTime = function (slug, from, to) {
...
+
+ApiClient_.prototype.fetchTopHoldersPercentOfTotalSupply = function (slug, from, to, numberOfHolders) {
+  var query = {
+    'query': '{\
+       topHoldersPercentOfTotalSupply(slug: "' + slug + '",\
+                                      from: "' + toUTC_(from) + '",\
+                                      to: "' + toUTC_(to) + '",\
+                                      interval: "1d",\
+                                      numberOfHolders: ' + numberOfHolders + ') {\
+                             datetime\
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: Indentation is strange here

File: src/api/queries/blockchainQueries.js [hunk 2]
```diff
@@ -242,3 +242,22 @@ ApiClient_.prototype.fetchEthSpentOverTime = function (slug, from, to) {
...
+                             inExchanges\
```
Comment (kalkov) [inline src/api/queries/blockchainQueries.js]: Better order would be in changes, outside exchanges, in top holders

File: src/api/queries/blockchainQueries.js (hunk 1 above)
Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: Is it alright now ?

File: src/api/queries/blockchainQueries.js (hunk 2 above)
Comment (spiderjako) [reply src/api/queries/blockchainQueries.js]: Just changed it, should be alright.

**Pull request merged**